src/cmake/common-settings.cmake

```diff
@@ -99,23 +99,21 @@ endif()
 
 
 if(MSVC)
-	#Compile option must be updated if VPCKG static is used (can't mix different compile option)
-	if("${VCPKG_TARGET_TRIPLET}" STREQUAL "x64-windows-static" OR "${VCPKG_TARGET_TRIPLET}" STREQUAL "x86-windows-static")
-		set(CMAKE_C_FLAGS_DEBUG   "${COMMON_MSVC_FLAGS} /MTd /GR /Ot /Od /EHsc /RTC1")
-		set(CMAKE_CXX_FLAGS_DEBUG "${COMMON_MSVC_FLAGS} /MTd /GR /Ot /Od /EHsc /RTC1 /fp:except")
-    else()
-		set(CMAKE_C_FLAGS_DEBUG   "${COMMON_MSVC_FLAGS} /MDd /GR /Ot /Od /EHsc /RTC1")
-		set(CMAKE_CXX_FLAGS_DEBUG "${COMMON_MSVC_FLAGS} /MDd /GR /Ot /Od /EHsc /RTC1 /fp:except")
-	endif()
+	set(CMAKE_C_FLAGS   "${COMMON_MSVC_FLAGS}")
```

Comment:
This ensures those flags will be used for all builds, including debug an release.
Reference to `COMMON_MSVC_FLAGS` in config-specific flags are removed hereafter, since they are redundant.